CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+#How to contribute
+TODO
```

Comment:
I was trying to rewrite the opening paragraph, but think it's unnecessary. Let's just jump straight into the project details link (and not worry about the todo here)

Reply:
Done